eng/WpfArcadeSdk/tools/CodeAnalysis.targets

```diff
@@ -6,8 +6,7 @@
   <ItemGroup Condition="'$(IsTestProject)'!='true' AND '$(EnableAnalyzers)'=='true'">
     <!-- Managed Code Reference analyzers -->
     <PackageReference Include="Microsoft.DotNet.CodeAnalysis" Version="$(MicrosoftDotNetCodeAnalysisPackageVersion)" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="2.9.0" />
```

Comment:
This shouldn't be needed and it conflicts with other PackageReference when enabling analyzers in any other project than System.Xaml.